src/routes/userRoutes.js

```diff
@@ -0,0 +1,9 @@
+import { Router } from "express";
+import userController from "../controllers/userController.js";
+import URL from "../navigation/pathes.js";
```

Comment:
Let's try to setup relative and absolute paths into aour application.
At this moment its pretty small, but for future uses it will be really helpful to get something without dots and file extension in path.

Example: 
```
import URL from "db/navigation/pathes";
import userController from "controllers/userController";
```